apps/condo/domains/notification/templates.js

```diff
@@ -1,135 +1,150 @@
-const conf = require('@core/config')
 const dayjs = require('dayjs')
 const get = require('lodash/get')
+const isFunction = require('lodash/isFunction')
+const isEmpty = require('lodash/isEmpty')
+
+const conf = require('@core/config')
 
 const { RU_LOCALE, EN_LOCALE, LOCALES } = require('@condo/domains/common/constants/locale')
 
 const {
+    MESSAGE_TYPES,
+    EMAIL_TRANSPORT,
+    SMS_TRANSPORT,
+    PUSH_TRANSPORT,
+    MESSAGE_TRANSPORTS,
     INVITE_NEW_EMPLOYEE_MESSAGE_TYPE,
     DIRTY_INVITE_NEW_EMPLOYEE_MESSAGE_TYPE,
-    MESSAGE_TRANSPORTS,
     REGISTER_NEW_USER_MESSAGE_TYPE,
     RESET_PASSWORD_MESSAGE_TYPE,
     SMS_VERIFY_CODE_MESSAGE_TYPE,
     SHARE_TICKET_MESSAGE_TYPE,
-    EMAIL_TRANSPORT,
-    SMS_TRANSPORT,
     DEVELOPER_IMPORTANT_NOTE_TYPE,
     CUSTOMER_IMPORTANT_NOTE_TYPE,
-    MESSAGE_FORWARDED_TO_SUPPORT,
+    MESSAGE_FORWARDED_TO_SUPPORT_TYPE,
+    TICKET_ASSIGNEE_CONNECTED_TYPE,
+    TICKET_EXECUTOR_CONNECTED_TYPE,
+    TICKET_STATUS_CHANGED_TYPE,
 } = require('./constants/constants')
 
-async function renderTemplate (transport, message) {
-    if (!MESSAGE_TRANSPORTS.includes(transport)) throw new Error('unexpected transport argument')
+const SERVER_URL = conf.SERVER_URL
 
-    // TODO(pahaz): we need to decide where to store templates! HArDCODE!
-    // TODO(pahaz): write the logic here!
-    //  1) we should find message template by TYPE + LANG
-    //  2) we should render the template and return transport context
+const getInviteNewEmployeeMessage = (message) => {
```

Comment:
You don't need to split it. it's better to use common logic for this. It's not a part of your task